project/build.properties

```diff
@@ -1,2 +1,2 @@
 # build was failing on sbt 1.0.2, so pin to sbt 0.13.
-sbt.version=0.13.16
+sbt.version=0.13.17
```

Comment:
I wanted the sbt version in the main codebase and in automation to be the same, so I upgraded both to 0.13.17.